layouts/shortcodes/codeimporter.html

```diff
@@ -1,8 +1,27 @@
 {{ $url := .Get "url" }}
 {{ $type := .Get "type" }}
-{{ with resources.GetRemote (urls.Parse $url) }}
-{{ $codeBlock := printf "```%s\n%s\n```" $type .Content }}
-{{ $codeBlock | markdownify }}
+{{ $startLine := .Get "startLine" | default 1 | int }}
+{{ $startLine = sub $startLine 1 }}
+{{ $endLine := .Get "endLine" | default -1 | int }}
+{{ $selectedLines := slice }}
+{{ with resources.GetRemote ( printf $url ) }}
```

Comment:
Replace `printf` with `urls.Parse` to handle encoded urls. (For context check https://github.com/nunocoracao/blowfish/issues/1549)